### DISCUSSION TITLE
βιογραφικό Β, local generation windows

### DISCUSSION THREAD

**Discussion by akimo13** [upatras-hci/iv#80] [Q&A] [answered]
Καλησπέρα @epidrome, για την περάτωση της δραστηριότητας 7 έχω κάνει fork το repository του [simple-cv](https://github.com/plain-plain-text/simple-cv). Κάνοντας edit στα αρχεία .md του φακέλου sections και τα .yml του φακέλου metadata έχω αρχίσει και διαμορφώνω το βιογραφικό σύμφωνα με τα δικά μου στοιχεία. Στη συνέχεια τρέχοντας την εντολή `sh process.sh` κάνω export σε .pdf  όπου και μπορώ και βλέπω τις αλλαγές που έχουν γίνει.

Να σημειώσω πως η όλη διαδικασία γίνεται από το Terminal του Visual Studio Code, σε περιβάλλον Windows, έχοντας εγκαταστήσει και το [MiKTeX](https://miktex.org/). Με το ArchLinux είχα αντιμετωπίσει κάποια προβλήματα, με τελευταίο την έλλειψη χώρου για εγκατάσταση των πακέτων του Pandoc. Στην προσπάθεια μου να ανακτήσω περισσότερο χώρο, το partition του ArchLinux διεγράφη.

Θα ήθελα λοιπόν να ρωτήσω αν η παραπάνω διαδικασία θεωρείται σωστή και καλύπτει τις απαιτήσεις της δραστηριότητας, όσον αφορά στην παραγωγή του αρχείου pdf.

**Comment by epidrome** [comment, marked as answer]
σίγουρα είναι στην σωστή κατεύθυνση, ιδανικά θα θέλαμε να παράγεται αυτόματα με GitHub action or Continuous Integration

γενικά οι ασκήσεις δεν έχουν σωστό-λάθος, αλλά βαθμό-επίπεδο ολοκλήρωσης, εκτιμώ ότι η προσπάθεια σας είναι κάπου στο 7/10

**Comment by akimo13** [comment]
Ευχαριστώ, συνέχισα με την παραμετροποίηση των .md και .yml αρχείων. Στη συνέχεια συνέδεσα το repository μου με το Netlify για να γίνει deploy. Κάθε φορά που γίνεται ένα νέο push στο repositrory μου του simple cv γίνεται εκ νέου build στο Netlify. Πρέπει όμως να σημειώσω πως για να παραχθούν τα απαραίτητα αρχεία, πρέπει κάθε φορά να τρέξω και την εντολή `sh. process.sh`.
